src/webportal/src/app/user/fabric/batchRegister/TopBar.jsx

```diff
@@ -37,7 +37,7 @@ function TopBar() {
     iconProps: {
       iconName: 'OpenFolderHorizontal',
     },
-    buttonStyles: {root: buttonPrimaryStyles},
+    buttonStyles: {root: [buttonMarginStyle, {marginLeft: '-8px'}]},
```

Comment:
I think here should be `-spacing.s1`.

Reply:
or maybe you can use IButtonStyle to clear the style.
https://developer.microsoft.com/en-us/fabric#/controls/web/button#IbuttonStyles

Reply:
Changed to use `-spacing.s1`.